lib/rules/require-capitalized-comments.js

```diff
@@ -52,7 +52,7 @@
  * ```
  *
  * ```js
- * "requireCapitalizedComments": { allExcept: ["jshint"] }
+ * "requireCapitalizedComments": { "allExcept": ["jshint"] }
```

Comment:
Isn't this exact comment used for the rules page? I looked at the website
source and came to the conclusion this file was used.
On Apr 30, 2015 1:13 PM, "Oleg Gaidarenko" <notifications@github.com> wrote:

> In lib/rules/require-capitalized-comments.js
> <https://github.com/jscs-dev/node-jscs/pull/1322#discussion_r29417745>:
>
> > @@ -52,7 +52,7 @@
> >   * ```
> >   *
> >   * ```js
> > - * "requireCapitalizedComments": { allExcept: ["jshint"] }
> > + * "requireCapitalizedComments": { "allExcept": ["jshint"] }
>
> Although it is not fully documented but we can accept config declared
> through js format
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jscs-dev/node-jscs/pull/1322/files#r29417745>.
>